app/models/following.rb

```diff
@@ -4,4 +4,12 @@ def initialize(following_data)
     @login = following_data[:login]
     @link = following_data[:html_url]
   end
+
+  def has_account?
```

Comment:
Is it preferable to have this method in both followers and followings or in one place elsewhere